tests/Vote.Monitor.Api.Feature.Observer.UnitTests/Specifications/ObserverAggregateFaker.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Vote.Monitor.Api.Feature.Observer.UnitTests.Specifications;
+
+public class ObserverAggregateFaker : AutoFaker<ObserverAggregate>
+{
+    UserStatus[] statuses = new UserStatus[] { UserStatus.Active, UserStatus.Deactivated };
+    public ObserverAggregateFaker(string? name=null, string? login=null, string? password=null, UserStatus? status=null)
+    {
+        RuleFor(fake => fake.Name, fake => name ?? fake.Name.FirstName());
+        RuleFor(fake => fake.Login, fake => login ?? fake.Internet.Email());
+        RuleFor(fake => fake.Password, fake => password ?? fake.Random.String2(10));
+        RuleFor(fake => fake.Status, fake => status ?? statuses[fake.Random.Int(0, 1)]);

```

Comment:
this is also a solution a "cleaner" will be ` fake.PickRandom(statuses)`

Reply:
great, thank you